Classes/OldModel/Event.php

```diff
@@ -4615,11 +4615,11 @@ public function createImageForSingleView($maxImageWidth, $maxImageHeight)
         $imageUrl = [];
         preg_match('/src="([^"]*)"/', $imageWithTag, $imageUrl);
 
-        return ' style="' .
-            'background-image: url(\'' . $imageUrl[1] . '\'); ' .
-            'background-repeat: no-repeat; ' .
-            'padding-left: ' . $imageWidth[1] . 'px; ' .
-            'height: ' . $imageHeight[1] . 'px;"';
+        return ' <img '.
+          'src=" '. $imageUrl[1]  .'" ' .
+          'style="width:'.$imageWidth.'px;height:'.$imageHeight.'px;"'.
+          '>';
+
```

Comment:
Instead of rebuilding the `img` tag, you can directly use `$imageWithTag`. Also please rename it to `$imageTag` as the purpose now has changed.

You'll also need to adapt the corresponding unit test that's currently failing: `Tx_Seminars_Tests_Unit_FrontEnd_DefaultControllerTest::testSingleViewDisplaysSeminarImage`